Corona-Warn-App/src/main/java/de/rki/coronawarnapp/eventregistration/checkins/download/CheckInsPackage.kt

```diff
@@ -27,41 +27,45 @@ private const val TYPE_EVENT = 2
 private val dummyEventCheckIn1: CheckIn = CheckIn(
     id = 1L,
     guid = "eventOne",
+    guidHash = ByteArray(0),
     version = 1,
     type = TYPE_LOCATION,
     description = "Restaurant",
     address = "Around the corner",
     traceLocationStart = null,
     traceLocationEnd = null,
     defaultCheckInLengthInMinutes = null,
+    byteRepresentation = ByteArray(0),
     signature = "signature",
     checkInStart = Instant.ofEpochMilli(
         DateTime(2021, 2, 2012, 11, 45).millis
     ),
     checkInEnd = Instant.ofEpochMilli(
         DateTime(2021, 2, 20, 12, 15).millis
     ),
-    targetCheckInEnd = null,
+    completed = false,
     createJournalEntry = false
 )
 
 private val dummyEventCheckIn2: CheckIn = CheckIn(
     id = 1L,
     guid = "eventOne",
+    guidHash = ByteArray(0),
     version = 1,
     type = TYPE_EVENT,
-    description = "Women in tech meetup",
+    description = "People in tech meetup",
```

Comment:
😄

Reply:
What kind of message do you want to send here? Women in Tech meetups exist because there is widespread discrimination against women.